app/vep/utils/vcf_results.py

```diff
@@ -66,13 +66,15 @@ def _get_prediction_index_map(
         if csq_headers[index] in target_columns
     }
 
-    # TODO add exception if len(map) != len(TARGET_COLUMNS)
+    #if len(map_index) != len(target_columns): #CANONICAL is often missing
+    #    raise Exception(f"Required columns missing from CSQ header: {map_index.keys()}")
```

Comment:
Are these two commented lines left intentionally, or forgotten?

Reply:
Itentional. I implemented the TODO comment, but it was a bad idea in practice: some missing columns (like "canonical" in my  test case) should not raise exception because it's recoverable (default values defined in the code). Needs decision how to proceed with the column check.